tooling/lsp/src/lib.rs

```diff
@@ -41,12 +45,20 @@ mod types;
 use solver::WrapperSolver;
 use types::{notification, request, NargoTest, NargoTestId, Position, Range, Url};
 
+#[derive(Debug, Error)]
+pub enum LspError {
+    /// Error while Resolving Workspace - {0}.
```

Comment:
```suggestion
    /// Error while resolving `Workspace`
```

I don't think that the `- {0}` does anything in the doc comment here.